### ISSUE TITLE
Create dedicated repo for additional terratest examples (e.g Lambda, SQS, Redshift, etc)

### ISSUE THREAD

**Issue by davegallant** [gruntwork-io/terratest#410]
I could not find any example code using AWS Lambda in the repo. Is there any reason why AWS Lambda and terratest would not work well together?

I've done some initial work in a fork here to see how it might work: https://github.com/davegallant/terratest/commit/f5fdaea97646e63696d83ec09e2f845ffb214f57

Do not want to contribute bloat to the repo, but AWS Lambda is a popular AWS service.

**Comment by yorinasub17** [comment]
AWS Lambda would work well as well, but at the same time, we don't have any specific functions for testing AWS Lambda.

The examples in the terratest repo serve two purposes: serve as examples for the internal functions that we provide in the modules, and as integration tests for those functions. In the case of AWS Lambda, since we don't have any specific functions that target these, we don't have any tests in the examples. I think we would like to keep it that way to avoid bloating the CI pipeline for terratest (each new example is another test we have to run everytime we update the library).

With that said, I think it makes sense to have more example usage of terratest. These should live in another repo. We currently don't have a home for this, but we can use this ticket for tracking that.

**Comment by Aboitier** [comment]
Yes, having more test functions targeting other services would be nice. Thanks for bringing Terratest alive!

**Comment by james03160927** [comment]
We do have a lot of examples in this folder: https://github.com/gruntwork-io/terratest/tree/main/test. Closing this issue but feel free to reopen it if you have any further concern or questions.
